frontend/src/metabase/parameters/components/ParameterWidget/ParameterWidget.jsx

```diff
@@ -59,6 +64,8 @@ export class ParameterWidget extends Component {
         commitImmediately={commitImmediately}
         setParameterValueToDefault={setParameterValueToDefault}
         enableRequiredBehavior={enableParameterRequiredBehavior}
+        isSortable={isSortable}
```

Comment:
Can we pass `isSortable={isSortable && !isEditing}` here to avoid adding `disableSort`? or something like that

Reply:
Yessir!